Arc/App/Models/ACState.swift

```diff
@@ -34,6 +34,7 @@ public enum ACState : String, State, CaseIterable {
 	static var surveys:[ACState] { return  [.context] }
 	
 	static var tests:[ACState] {return [.gridTest, .priceTest, .symbolsTest] }
+    static public var totalTestCountInSession = 3
```

Comment:
On some screens, the library assumed all 3 assessments would run at once.  This variable can be changed so that totalTestCountInSession = 1 to change the verbiage and behavior on some screens to support single assessment test session runs.